docs/visual-basic/developing-apps/windows-forms/how-to-change-the-layout-of-a-datarepeater-control-visual-studio.md

```diff
@@ -47,7 +47,7 @@ The <xref:Microsoft.VisualBasic.PowerPacks.DataRepeater> control can be displaye
     > [!NOTE]
     >  You must select the outer border of the <xref:Microsoft.VisualBasic.PowerPacks.DataRepeater> control by clicking in the lower region of the control, not in the upper <xref:Microsoft.VisualBasic.PowerPacks.DataRepeater.ItemTemplate%2A> region.  
   
-2.  In the Properties window, set the <xref:Microsoft.VisualBasic.PowerPacks.DataRepeater.LayoutStyle%2A> property to either <xref:Microsoft.VisualBasic.PowerPacks.DataRepeaterLayoutStyles.Vertical> or <xref:Microsoft.VisualBasic.PowerPacks.DataRepeaterLayoutStyles.Horizontal>.  
+2.  In the Properties window, set the <xref:Microsoft.VisualBasic.PowerPacks.DataRepeater.LayoutStyle%2A> property to either `DataRepeaterLayoutStyles.Vertical` or `DataRepeaterLayoutStyles.Horizontal`.  
```

Comment:
instead of changing the xref link, shouldn't you instead add those entries into https://github.com/dotnet/docs/blob/master/_zip/missingapi.yml? BTW, I'll start an email thread on the dv_vboPrinterref project. We probably need to migrate these APIs no?